apps/api/src/view-models/buildViewModelForResource/viewModels/utilities/graph/__snapshots__/buildTreeFromNodes.spec.ts.snap

```diff
@@ -0,0 +1,173 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`buildTreeFromNodes when given a valid tree should return the expected result 1`] = `
```

Comment:
It might be nice to have some actual check that the nested structure is true to the original array representation's hierarchy.

Reply:
You **can** collapse the various levels of the snapshot and see that the hierarchy is correct.